xds/src/main/java/com/linecorp/armeria/xds/client/endpoint/XdsEndpointGroup.java

```diff
@@ -92,114 +86,125 @@ public static EndpointGroup of(ListenerRoot listenerRoot) {
     @UnstableApi
     public static EndpointGroup of(ClusterSnapshot clusterSnapshot) {
         requireNonNull(clusterSnapshot, "clusterSnapshot");
-        return new XdsEndpointGroup(clusterSnapshot);
+        return new XdsEndpointGroup(new ClusterManager(clusterSnapshot), false);
```

Comment:
Did you intentionally not add the method that accepts `allowEmptyEndpoints`?

Reply:
I didn't want to add APIs if I could help it. For this case, I think this API is likely only used by `XdsBootstrap`. Let me know if you feel differently though

Reply:
Just wanted to make sure if you missed it or not. 😉